src/Components/CreateImageWizardV2/steps/Packages/PackageRecommendations.tsx

```diff
@@ -112,7 +112,7 @@ const PackageRecommendations = () => {
             {isSuccess && !data && (
               <>No recommendations found for the set of selected packages</>
             )}
-            {isSuccess && data && data[0].packages && (
+            {isSuccess && data && data[0]?.packages && (
```

Comment:
Couldn't this just be `isSuccess && data[0].packages && ...`?

Reply:
Not sure if this PR is up-to-date as API schema changed in the morning (https://github.com/osbuild/image-builder/pull/1165). Fix for this bug is a part of https://github.com/osbuild/image-builder-frontend/pull/2024

Reply:
I have the included fix in my branch if you want to close this.

Reply:
we can close this pr, because the API schema changed

Reply:
and this fix appears in @Andrewgdewar's pr